lib/rom/support/inflector.rb

```diff
@@ -13,8 +13,11 @@ module Inflector
           ::Inflecto
         end
     rescue LoadError
-      raise 'Unable to find an inflector library'
+      raise LoadError,
+            "No inflector library could be found: "\
+            "please install either the `activesupport` or `inflecto` gem."
```

Comment:
Hehe, fair enough!  Do you think we should prioritise `inflecto` in the load ordering too?  I don't much mind either way! :smiley_cat: